api/app/query.py

```diff
@@ -0,0 +1,55 @@
+from typing import List, Tuple
+
+
+def create_gadm_dist_query(
```

Comment:
I think a follow up activity would be to see how this works using the [builder pattern](https://refactoring.guru/design-patterns/builder)